showcase/src/app/demo/slider/slider.component.html

```diff
@@ -0,0 +1,36 @@
+<example-viewer [extensions]="['HTML', 'TS']" 
+				filesPath="slider/simple-slider"
+				title="SHOWCASE.DEMO.SLIDER.SIMPLE_SLIDER">
+	<div>
+		<h2 translate>SHOWCASE.DEMO.SLIDER.SIMPLE_SLIDER</h2>
+		<stark-slider [values]="simpleSliderValues"
+					  sliderId="simple-slider"
+					  [sliderConfig]="simpleSliderConfig"
+					  (changed)="onValuesChangeSimpleSlider($event)">
+		</stark-slider>
+		<div>
+			<label>Value:</label>
+			<input type="number" [value]="simpleSliderValues[0]" (change)="onChangeSimpleSlider($event)">
+		</div>
+	</div>
+</example-viewer>
+<example-viewer [extensions]="['HTML', 'TS']"
+				filesPath="slider/range-slider"
+				title="SHOWCASE.DEMO.SLIDER.RANGE_SLIDER">
+	<div>
+		<h2 translate>SHOWCASE.DEMO.SLIDER.RANGE_SLIDER</h2>
+		<stark-slider [values]="rangeSliderValues"
+					  sliderId="range-slider"
+					  [sliderConfig]="rangeSliderConfig"
+					  (changed)="onValuesChangeRangeSlider($event)">
```

Comment:
The "isDisabled" property is missing here. We should also demo such feature as in the old showcase ;)